account_mtd/models/mtd_construct_display_message.py

```diff
@@ -18,6 +18,8 @@ def construct_error_message_to_display(self, url=None, code=None, response_token
         if 'error' in response_token.keys():
             resp_error = response_token['error']
         resp_error_message = ""
+        if 'warnings' in response_token.keys():
```

Comment:
Do you need to iterate over "warnings" the same way you do with "errors" to build the message?

Reply:
Spoke to @Parikshit-Vaghasiya about this. No need to iterate the warnings